ui/src/overview.tsx

```diff
@@ -23,24 +24,41 @@ import {
   renameCriteria,
   renameGroup,
 } from "cohortsSlice";
+import Loading from "components/loading";
 import { useMenu } from "components/menu";
 import { useTextInputDialog } from "components/textInputDialog";
+import { useAsyncWithApi } from "errors";
 import { useAppDispatch, useCohort, useUnderlay } from "hooks";
-import React from "react";
+import { useCallback, useContext } from "react";
 import { Link as RouterLink, useHistory } from "react-router-dom";
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
 import { createUrl } from "router";
 import * as tanagra from "tanagra-api";
-import { createCriteria, getCriteriaPlugin } from "./cohort";
+import {
+  createCriteria,
+  generateQueryFilter,
+  getCriteriaPlugin,
+} from "./cohort";
 
 export default function Overview() {
   const cohort = useCohort();
+  const underlay = useUnderlay();
 
   return (
     <>
       <ActionBar title={cohort.name} />
       <Grid container columns={3} columnSpacing={5} className="overview">
         <ParticipantsSelector kind={tanagra.GroupKindEnum.Included} />
         <ParticipantsSelector kind={tanagra.GroupKindEnum.Excluded} />
+        <DemographicCharts cohort={cohort} underlayName={underlay.name} />
```

Comment:
Call useUnderlay in DemographicChartProps. There's no need for it to be a parameter.